code/game/atoms_movable.dm

```diff
@@ -555,7 +557,8 @@
 		pixel_x_diff = -8
 
 	animate(src, pixel_x = pixel_x + pixel_x_diff, pixel_y = pixel_y + pixel_y_diff, time = 2)
-	animate(pixel_x = initial(pixel_x), pixel_y = final_pixel_y, time = 2)
+	//animate(pixel_x = initial(pixel_x), pixel_y = final_pixel_y, time = 2) //CITADEL CHANGE, DOGBORG OFFSET

```

Comment:
you can remove this line instead of simply commenting it out, and add //CITADEL CHANGE - replaces pixel_x = initial(pixel_x) with pixel_x = pixel_x - pixel_x_diff

Reply:
The commenting out is an additional layer of 'just in case' and also a reminder to try and push this upstream